qemu/tests/cfg/block_with_share_rw.cfg

```diff
@@ -39,11 +39,11 @@
                             drive_format_stg = "usb3"
         -@passthrough:
             only virtio_scsi
-            pre_command = "modprobe -r scsi_debug; modprobe sg; "
-            pre_command += "modprobe scsi_debug add_host=1 dev_size_mb=50"
+            start_vm = no
+            check_image_cmd = "ls /dev/ | grep sda"
+            modprobe_cmd = "modprobe -r scsi_debug; modprobe sg; "
+            modprobe_cmd += "modprobe scsi_debug add_host=1 dev_size_mb=50"
             post_command = "rmmod scsi_debug"
```

Comment:
Why put those options into py file ? Does it have problem if Keep them in cfg file

Reply:
Yes, if pre_command is used, the running of this cmd will run before the checkpoint.

Reply:
I mean image_raw_device_stg and indirect_image_select_stg

Reply:
yes, https://github.com/avocado-framework/avocado-vt/blob/master/virttest/storage.py#L283

ERROR: Matching disk is in blacklist. name = '/dev/sg*', matching = '['', '/dev/sg0']' and selector = '-1'